src/components/Board.js

```diff
@@ -28,22 +29,25 @@ const Board = (props) => {
   }, [props.boardName]);
 
   const cardComponents = () => {
-    return cards.map((card) => {
+    return cards.map((cardh) => {
+      console.log(cardh)
       return (
-        <section key={card.id}>
-          <Card id={card.id} text={card.text} emoji={card.emoji} />
+        <section key={cardh.card.id}>
+          <Card id={cardh.card.id} text={cardh.card.text} emoji={cardh.card.emoji} />
         </section>
       );
     });
   };
+
   return (
     <div className="validation-errors-display">
       {errorMessage && (
         <div className="validation-errors-display_list">
           <h2>{errorMessage}</h2>
         </div>
       )}
-      <section className="board">{cardComponents}</section>
+      {/* add () to call the fuction, without () is only referencing it */}
+      <section className="board">{cardComponents()}</section> 
```

Comment:
You got this right, just needed to add () to make sure we call the function rather than just referencing it.